tofu/tests/tests00_root/test_03_plot.py

```diff
@@ -10,17 +10,14 @@
 import matplotlib.pyplot as plt
 import warnings as warn
 
-# Nose-specific
-from nose import with_setup # optional
```

Comment:
Are w sure this line was not used somewhere ?

Reply:
Yes, I think it was something old your tried, it was used in comments. I took them all out in the latest push